esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2298,6 +2264,66 @@ bool CFileSprayEx::onSprayVariable(IEspContext &context, IEspSprayVariable &req,
     return true;
 }
 
+void CFileSprayEx::checkDZScopeAccessAndSetSpraySourceDFUFileSpec(IEspContext &context, const char *srcPlane, const char *srcHost,
+    const char *srcFile, MemoryBuffer &srcXML, IDFUfileSpec *srcDFUfileSpec)
+{
+    if(srcXML.length() == 0)
+    {
+        RemoteMultiFilename rmfn;
```

Comment:
trivial: would be better to move this declaration to closer to where 1st used below.

Reply:
Will change